processing/data_collection/gazette/spiders/rn_natal.py

```diff
@@ -11,15 +12,23 @@ class RnNatalSpider(BaseGazetteSpider):
 
     name = "rn_natal"
     allowed_domains = ["www.natal.rn.gov.br"]
+    start_date = date(2003, 1, 1)
```

Comment:
I guess it should be `"2003-1-1"` so you can parse the input string in `start_requests`, otherwise you won't be able to set it through the CLI

Reply:
My bad... :sweat: